src/Slider/types.ts

```diff
@@ -18,6 +18,12 @@ export type SliderProps = {
   children: JSX.Element[];
   sliderUpdates?: sliderUpdateType[];
   defaultSpaceBetween?: number;
+  showDots?: boolean;
+  customDot?: JSX.Element;
+  customActiveDot?: JSX.Element;
+  defaultDotColor?: string;
+  defaultDotActiveColor?: string;
```

Comment:
`activeColorForDefaultDot`  color for default dot but not defaul color for dot

rewrite for other dots` name